homeassistant/components/synology_srm/device_tracker.py

```diff
@@ -37,6 +37,34 @@
     }
 )
 
+ATTRIBUTE_ALIAS = {
+    "band": None,
+    "connection": None,
+    "current_rate": None,
+    "dev_type": None,
+    "hostname": None,
+    "ip6_addr": None,
+    "ip_addr": None,
+    "is_baned": None,
+    "is_beamforming_on": None,
+    "is_guest": None,
+    "is_high_qos": None,
+    "is_low_qos": None,
+    "is_manual_dev_type": None,
+    "is_manual_hostname": None,
+    "is_online": None,
+    "is_parental_controled": None,
```

Comment:
`is_parental_controled` -> `is_parental_controlled`

Reply:
Same with the question above... will await the answer on that